src/createBot.test.ts

```diff
@@ -235,8 +244,11 @@ describe('server functions', () => {
 
   test('verify webhook token', async () => {
     const challenge = 'random';
+    log(`${webhookPath}?hub.mode=subscribe&hub.verify_token=${encodeURIComponent(webhookVerifyToken)}&hub.challenge=${challenge}`);
```

Comment:
Nit: let's create a variable called `url` & reuse this in both the log & get function